cogs/StaffAppsUser.py

```diff
@@ -0,0 +1,567 @@
+        """
+        return f"User: {self.uid}, Joined: {self.joined_at}, Messages: {self.messages_sent}, Marked Spam: {self.marked_spam}, Cooldown: {self.cooldown}"
+
+    def __repr__(self) -> str:
+        """
+        Same as __str__.
+        """
+        return f"User: {self.uid}, Joined: {self.joined_at}, Messages: {self.messages_sent}, Marked Spam: {self.marked_spam}, Cooldown: {self.cooldown}"
+
+    def min_reqs(self) -> tuple[bool, int, datetime]:
+        """
+        Returns a tuple of (bool, int, datetime) where bool is True if the user meets the minimum requirements to apply for staff.
+        """
+        time_since_join = datetime.now(tz=self.joined_at.tzinfo) - self.joined_at
+        if (
+            self.joined_at
+            <= (datetime.now(tz=self.joined_at.tzinfo) - timedelta(days=30))
+            and self.messages_sent >= 500
+        ):
+            return (True, self.messages_sent, time_since_join)
+        return (False, self.messages_sent, time_since_join)
+
+    def is_on_cooldown(self) -> bool:
+        """
+        Returns True if the user is on cooldown.
+        """
+        return (self.cooldown - time() // 1) > 0
+
+
+class StaffAppView(discord.ui.View):
+    def __init__(self, author: user, bot: commands.Bot) -> None:
+        """
+        A discord view for the staff application.
+        Includes a select menu for:
+            - NDA
+            - Timezone
+            - Hours Available
+        """
+        super().__init__()
+        self.author = author
+        self.bot = bot
+
+    @discord.ui.select(
+        placeholder="Do you agree to the Non-Disclosure Agreement?",
+        row=0,
+        min_values=1,
+        max_values=1,
+        options=[
+            discord.SelectOption(
+                label="I Agree",
+                description="I agree to the Non-Disclosure Agreement.",
+                emoji="✅",
+            ),
+            discord.SelectOption(
+                label="I do NOT Agree",
+                description="I do not agree to the Non-Disclosure Agreement.",
+                emoji="❌",
+            ),
+        ],
+    )
+    async def nda_callback(
+        self,
+        select: discord.ui.Select,
+        interaction: discord.Interaction,
+    ) -> None:
+        """
+        If the user does not agree to the NDA, the application is cancelled.
+        """
+        answer = select.values[0]
+        if answer == "I do NOT Agree":
+            embed = discord.Embed(
+                title="Staff Application",
+                description="You have not agreed to the Non-Disclosure Agreement.\n Unfortunately, we cannot complete your staff application without your agreement. Please re-apply when you are ready to agree to the Non-Disclosure Agreement.\n *(You can re-apply by running this command again)*",
+                color=0xFF0000,
+            )
+            await interaction.response.edit_message(embed=embed, view=None)
+            return
+        self.nda = True
```

Comment:
`__init__`?

Reply:
`self.nda = True` is part of the callback when a user selects this option. It should not be included in the init